### ISSUE TITLE
Praat interactive shell broken

### ISSUE THREAD

**Issue by psibre** [praat/praat#199]
According to the Manual (_Scripting 6.9_), running `praat -` should enter an interactive shell, or allow praat to accept commands from STDIN.
This worked up to v5.4.14:

```
$ /Volumes/Praat64_5414/Praat.app/Contents/MacOS/Praat -
Praat > echo Hello
Hello
Praat > Quit
$ 
```

However, from v5.4.15 onwards, this no longer works:

```
$ /Volumes/Praat64_5415/Praat.app/Contents/MacOS/Praat -
^C
```

It just hangs, and the `Praat` prompt never appears.

**Comment by jjatria** [comment]
I can confirm this to also be the case (same version numbers) in Linux (Kubuntu 15.10 32-bit).

**Comment by dlukes** [comment]
This is still the case. I think it's a bug in the (then) new command line parsing code introduced in v5.4.15:

https://github.com/praat/praat/blob/v5.4.15/sys/praat.cpp#L1057-L1059

The branch that matches the `-` command line arg never increments `iarg_batchName`, so the while loop never terminates. I have a fix that seems to work in dcaf03d, I'll open a PR :)
